pdg/src/snapshots/c2rust_pdg__tests__analysis_test_pdg_snapshot.snap

```diff
@@ -165,59 +165,59 @@ g {
 	n[1]:  copy n[0]  => _7  @ bb2[14]:  fn exercise_allocator;      _7 = move _8 as *const u8 (Pointer(ArrayToPointer))  
 	n[2]:  copy n[1]  => _6  @ bb2[16]:  fn exercise_allocator;      _6 = move _7 as *const i8 (Misc)                     
 	n[3]:  copy n[2]  => _1  @ bb0[0]:   fn printf;                  _5 = pointers::printf(move _6, move _10)             
-	n[4]:  copy n[0]  => _30 @ bb11[7]:  fn exercise_allocator;      _30 = &raw const (*_31)                              
+	n[4]:  copy _     => _30 @ bb11[7]:  fn exercise_allocator;      _30 = &raw const (*_31)                              
```

Comment:
Does this have no source because it's a copy of a string literal?

Reply:
it's because the rvalue of the statement that assigns to `_31` isn't yet instrumented, i.e. `Operand::Constant`. I'll make an issue for this

Reply:
https://github.com/immunant/c2rust/issues/577